stable/dask-distributed/templates/dask-worker-deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "worker-fullname" . }}
+  labels:
+    heritage: {{.Release.Service | quote }}
+    release: {{.Release.Name | quote }}
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+    component: "{{.Release.Name}}-{{.Values.Worker.Component}}"
+spec:
+  replicas: {{default 1 .Values.Worker.Replicas}}
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      component: "{{.Release.Name}}-{{.Values.Worker.Component}}"
+  template:
+    metadata:
+      labels:
+        heritage: {{.Release.Service | quote }}
+        release: {{.Release.Name | quote }}
+        chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+        component: "{{.Release.Name}}-{{.Values.Worker.Component}}"
+    spec:
+      containers:
+        - name: {{ template "worker-fullname" . }}
+          image: "{{.Values.Worker.Image}}:{{.Values.Worker.ImageTag}}"
+          command: ["dask-worker", "{{ template "scheduler-fullname" . }}:{{.Values.Scheduler.ServicePort}}"]
+          ports:
+            - containerPort: {{.Values.Worker.ContainerPort}}
+          resources:
```

Comment:
I'd suggest you add resources like so: 
https://github.com/kubernetes/charts/blob/master/stable/kube-lego/templates/deployment.yaml#L49